### PR TITLE
Add toggle to display 5G as the recommended option in the preferred network type

### PR DESCRIPTION
**Pull request by ponces**

### CODE REVIEW

File: phh-prop-handler.sh [hunk 1]
```diff
@@ -257,3 +257,17 @@ if [ "$1" == "persist.bluetooth.system_audio_hal.enabled" ]; then
...
+    if [[ "$prop_value" != "0" && "$prop_value" != "1" ]]; then
+        exit 1
+    fi
+
+    if [[ "$prop_value" == 1 ]];then
+        resetprop_phh ro.telephony.default_network 33,33
+    else
+        resetprop_phh ro.telephony.default_network ${ro.vendor.radio.default_network}
+    fi
+    restartAudio
```
Comment (phhusson) [inline phh-prop-handler.sh]: Uh no need to restart audio?

File: phh-prop-handler.sh [hunk 2]
```diff
@@ -257,3 +257,17 @@ if [ "$1" == "persist.bluetooth.system_audio_hal.enabled" ]; then
...
+
+if [ "$1" == "persist.sys.phh.force_display_5g" ];then
+    if [[ "$prop_value" != "0" && "$prop_value" != "1" ]]; then
+        exit 1
+    fi
+
+    if [[ "$prop_value" == 1 ]];then
+        resetprop_phh ro.telephony.default_network 33,33
+    else
+        resetprop_phh ro.telephony.default_network ${ro.vendor.radio.default_network}
```
Comment (phhusson) [inline phh-prop-handler.sh]: Not all devices have that prop though. Some will directly set it in build.prop, some will set it from some shell script, some will set it from rc script, ...
I'd rather have this option require reboot, and in rw-system.sh have `if [ "$(getprop persist...)" = 1 ];then resetprop_phh ...;fi`

Comment (ponces) [reply phh-prop-handler.sh]: Makes sense. Will do. I need to update TrebleApp toggle summary then.
~Would it make sense to move this to rw-system then?~ nevermind, i'll do it on vndk.rc only

**Comment by phhusson** [comment]
FWIW, there is the small issue of single-sim or triple-sim devices. But let's consider it only if we hit this issue

**Pull request merged**